CelesteNet.Shared/DataTypes/DataChat.cs

```diff
@@ -22,9 +22,33 @@ public class DataChat : DataType<DataChat> {
         /// <summary>
         /// Server-internal field.
         /// </summary>
+        [Obsolete("Use TryGetSingleTarget instead of this getter, do `Targets = [value]` instead of this setter.", false)]
```

Comment:
This PR also has the change of obsoleting "Target" in DataChat in favor of only ever working with the "Targets" field.